frontend/src/app/testQueue/AoEForm/__snapshots__/AoEModalForm.test.tsx.snap

```diff
@@ -1,37 +1,122 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`AoEModalForm on data loaded renders 1`] = `
-<div>
-  <div
-    class="display-flex flex-justify"
+Object {
+  "asFragment": [Function],
+  "baseElement": <body
+    class="ReactModal__Body--open"
   >
-    <h1
-      class="font-heading-lg margin-top-05 margin-bottom-0"
-    >
-      Jobs, Steve
-    </h1>
     <div
-      class="sr-time-of-test-buttons"
+      data-react-modal-body-trap=""
+      style="position: absolute; opacity: 0;"
+      tabindex="0"
+    />
+    <div />
+    <div
+      data-react-modal-body-trap=""
+      style="position: absolute; opacity: 0;"
+      tabindex="0"
+    />
+    <div
+      class="ReactModalPortal"
     >
-      <button
-        class="modal__close-button"
-        style="cursor: pointer;"
+      <div
+        class="ReactModal__Overlay prime-modal-overlay display-flex flex-align-center flex-justify-center"
       >
-        <img
-          alt="Close"
-          class="modal__close-img"
-          src="close.svg"
-        />
-      </button>
+        <div
+          aria-label="Test questionnaire"
+          aria-modal="true"
+          class="ReactModal__Content"
+          role="dialog"
+          style="position: initial; top: 40px; left: 40px; right: 40px; bottom: 40px; border: 1px solid #ccc; background: rgb(255, 255, 255); overflow: auto; border-radius: 4px; outline: none; padding: 20px;"
+          tabindex="-1"
+        >
+          <div
+            class="display-flex flex-justify"
+          >
+            <h1
+              class="font-heading-lg margin-top-05 margin-bottom-0"
+            >
+              Jobs, Steve
+            </h1>
+            <div
+              class="sr-time-of-test-buttons"
+            >
+              <button
+                class="modal__close-button"
+                style="cursor: pointer;"
+              >
+                <img
+                  alt="Close"
+                  class="modal__close-img"
+                  src="close.svg"
+                />
+              </button>
+            </div>
+          </div>
+          <div
+            class="border-top border-base-lighter margin-x-neg-205 margin-top-1"
+          />
+          <h2
+            class="font-heading-lg margin-top-205 margin-bottom-0"
+          >
+            Test questionnaire
+          </h2>
+        </div>
+      </div>
     </div>
-  </div>
-  <div
-    class="border-top border-base-lighter margin-x-neg-205 margin-top-1"
-  />
-  <h2
-    class="font-heading-lg margin-top-205 margin-bottom-0"
-  >
-    Test questionnaire
-  </h2>
-</div>
+  </body>,
```

Comment:
Do you know why these lines were added? I haven't seen this being added before, so wondering what it does.

Reply:
I changed the test so the snapshot is now using the entire `RenderResult`, instead of just the `RenderResult.container`, so these are the other fields on the object. I had to do this when I removed the mock for `react-modal` because the modal component gets injected into a different part of the page and `container` is now just an empty div and what we really want to see is the `baseElement` field

it seems like most of our snapshots (at least the ones that use modals) include the whole object